packages/zoe/test/unitTests/contracts/test-multipoolAutoswap.js

```diff
@@ -45,14 +45,10 @@ test('multipoolAutoSwap with valid offers', async t => {
     const { invite: aliceInvite } = await zoe.makeInstance(
       installationHandle,
       harden({ CentralToken: centralR.issuer }),
-      harden({ CentralToken: centralR.issuer }),
     );
 
     const makeAmountMathFromIssuer = issuer =>
-      Promise.all([
-        E(issuer).getBrand(),
-        E(issuer).getMathHelpersName(),
-      ]).then(([brand, mathName]) => makeAmountMath(brand, mathName));
+      makeAmountMath(issuer.getBrand(), issuer.getMathHelpersName());
```

Comment:
I think we don't want to change this - the tests are more life-like when we assume things are remote even though we know they are aren't.